docs/PROI/report.tex

```diff
@@ -45,6 +45,90 @@ \subsubsection{Stretch Goals}
 
 \section{Machbarkeitsstudie}
 
+\subsection{NFC Tag}
+\subsubsection{lesen}
+Um mit NFC Tags arbeiten zu können, müssen diese auch auf entschlüsselt bzw. gelesen werden können.
+Hierfür ist ein Hardware NFC-Reader notwendig. Dieser kann über USB mit dem Chip der Box verbunden werden, um von dieser ausgelesen zu werden.
+\colorbox{yellow}{welche library wird genutzt?}
+
+\subsubsection{schreiben}
+Einen NFC-Tag zu beschreiben ist davon abhängig, ob der NFC-Tag über einen eigenen Speicher verfügt.
+Ist dieser vorhanden, können dort z.b. Kontaktdaten hinterlegt werden. Werden diese dann von einem Smartphone gelesen, öffnet sich die Kontakte-App und der auf dem NFC-Tag gespeicherte Kontakt kann abgespeichert werden.
+\\
+\colorbox{yellow}{Möglich, kann die gleiche library wie zum lesen genutzt werden?}
```

Comment:
Hatte schon @anbraten hier im PR angefragt, da ich nicht weiß wie ihr das am Ende gelöst habt, bzw welche library genutzt wird. Kann aber an sich aus der study raus denke ich, da das zu detailliert ist (?)

Reply:
The system we use is called evdev which is a kernel module of linux. It can handle some kind of event listening which is mainly used by keyboards. To use evdev with node we use [node-evdev](https://github.com/sdumetz/node-evdev) with TS and raspberry-pi support added by my [fork](https://github.com/anbraten/node-evdev).

Reply:
resolved with commit 6dea9fa2bae1760d5a6695423c4fbe54152beab4